include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -413,7 +413,7 @@ template <typename _ExecutionPolicy, typename _Range1, typename _Range2, typenam
           typename _LocalScan, typename _GroupScan, typename _GlobalScan>
 oneapi::dpl::__internal::__enable_if_device_execution_policy<
     _ExecutionPolicy, ::std::pair<oneapi::dpl::__internal::__difference_t<_Range2>, typename _InitType::__value_type>>
-__parallel_transform_scan(_ExecutionPolicy&& __exec, _Range1&& __rng1, _Range2&& __rng2, _BinaryOperation __binary_op,
+__parallel_transform_scan(_ExecutionPolicy&& __exec, _Range1&& __rng1, _Range2&& __rng2, _BinaryOperation,
```

Comment:
Do we need `_BinaryOperation`? Can we get rid of this argument?

Reply:
We need `_BinaryOperation` because there are two implementation of `__parallel_transform_scan`: in SYCL backend for FPGA and in common SYCL backend. This parameter is used in first one. Have I answered?

Reply:
Actually, FPGA backend is just call this one for now. So, if this function doesn't require `_BinaryOperation` then FPGA backend shouldn't require it too